str0m/src/media/mline.rs

```diff
@@ -991,6 +1001,55 @@ impl MLine {
         }
     }
 
+    /// Return the queue state for outbound RTP traffic for this mline.
+    ///
+    /// For the purposes of pacing each mline is treated as its own packet queue. Internally this
+    /// is spread across many [`PacketizingBuffer`] which is where the actual packets are queued.
+    pub fn send_queue_state(&self, now: Instant) -> QueueState {
```

Comment:
I think to be consistent with the naming, this would be better to tie into `buffer_tx`. Something like `buffer_tx_queue_state`.

Reply:
Sounds good